docs/Lua API.rst

```diff
@@ -1004,6 +1004,22 @@ Fortress mode
   Resets the fortress mode sidebar menus and cursors to their default state. If
   ``pause`` is true, also pauses the game.
 
+* ``dfhack.gui.pauseRecenter(pos[,pause])``
+  ``dfhack.gui.pauseRecenter(x,y,z[,pause])``
+
+  Same as ``resetDwarfmodeView``, but also recenter if ``x`` isn't ``-30000``,
+  and respects RECENTER_INTERFACE_SHUTDOWN_MS in DF's init.txt.
+
+* ``dfhack.gui.recenterViewscreen(pos[,zoom])``
+  ``dfhack.gui.recenterViewscreen(x,y,z[,zoom])``
+  ``dfhack.gui.recenterViewscreen([zoom])``
+
+  Recenter the view on a position using a specific zoom type. If no position is
+  given, recenter on ``df.global.cursor``. Zoom types are ``df.report_zoom_type``
+  (0 = Generic, 1 = Item, 2 = Unit), where Generic skips recentering and
+  enforces valid view bounds (the same as x = -30000.)
+  Default zoom type is Item.
```

Comment:
How do `Item` and `Unit` behave?

Reply:
A link to existing docs would work here too if the behavior is already documented somewhere